lib/build.sh

```diff
@@ -58,14 +58,21 @@ install_or_reuse_node_modules() {
   local build_dir=$1
   local layer_dir=$2
   local local_lock_checksum
+  local cached_lock_checksum
+
+  if ! detect_package_lock "$build_dir"; then
+    install_modules "$build_dir" "$layer_dir"
+    return 0
+  fi
+
+  touch "$layer_dir.toml"
 
   local_lock_checksum=$(sha256sum "$build_dir/package-lock.json" | cut -d " " -f 1)
+  cached_lock_checksum=$(yj -t < "$layer_dir.toml" | jq -r ".metadata.package_lock_checksum")
 
   mkdir -p "${layer_dir}"
 
-  if [[ -f "$build_dir/package-lock.json"
-    && -f "$layer_dir.toml"
-    && $local_lock_checksum == $(yj -t < "$layer_dir.toml" | jq -r ".metadata.package_lock_checksum") ]] ; then
+  if [[ -f "$layer_dir.toml" && "$local_lock_checksum" == "$cached_lock_checksum" ]] ; then
```

Comment:
is `-f "$layer_dir.toml"` necessary (see comment above)?

Reply:
You're right - we don't need this anymore